src/index.html

```diff
@@ -3,12 +3,97 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <nav class="nav__logo">
+        <a href="#home">
+          <img
+            class="logo"
+            src="./images/logo.png"
+            alt="logo"
+          >
+        </a>
+      </nav>
+
+      <div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item is-active">
+              <a
+                href="apple"
+                class="nav__link is-active"
+              >
+                apple
```

Comment:
```suggestion
                Apple
```
capital first is better